src/decorators/Model.ts

```diff
@@ -1,13 +1,20 @@
 const caller = require('caller'); // eslint-disable-line @typescript-eslint/no-var-requires
 import * as path from 'path';
 import { ObjectType } from 'type-graphql';
-import { Entity } from 'typeorm';
+import { ObjectOptions } from 'type-graphql/dist/decorators/ObjectType.d';
+import { Entity, EntityOptions } from 'typeorm';
 
 import { ClassType } from '../core';
 import { getMetadataStorage } from '../metadata';
 import { ClassDecoratorFactory, composeClassDecorators, generatedFolderPath } from '../utils/';
 
-export function Model() {
+interface ModelOptions {
+  api?: ObjectOptions;
+  db?: EntityOptions;
+}
+
+// Allow default TypeORM and TypeGraphQL options to be used
+export function Model({ api = {}, db = {} }: ModelOptions = {}) {
```

Comment:
@goldcaddy77 LMK if this seems to align with your thoughts.

Reply:
This looks great. I'll probably add some shortcuts at some point but this would unblock you